### ISSUE TITLE
it is possible to use exact location for HTML Markers?

### ISSUE THREAD

**Issue by Rociol84** [vasturiano/three-globe#69]
Hi! 
I would like to know if is possible to add specific location to the HTML Markers? right know looks like random locations. But I would like to add only two?
How can I do that?

thanks Rocio 

<img width="642" alt="Screenshot 2023-01-11 at 23 08 25" src="https://user-images.githubusercontent.com/122491828/211928407-8bfdf1f4-762e-4870-b43a-5efcaa788baa.png">

**Comment by vasturiano** [comment]
@Rociol84 yes you can use the `htmlLat` and `htmlLng` accessor methods to specify your coordinates. See more in [the docs](https://github.com/vasturiano/three-globe#html-elements-layer).
